templates/frontend/pages/catalogSeriesIndex.tpl

```diff
@@ -0,0 +1,66 @@
+{**
+ * templates/frontend/pages/catalogSeries.tpl
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University Library
+ * Copyright (c) 2003-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @brief Display the page to view books in a series in the catalog.
+ *
+ * @uses $series Series Current series being viewed
+ * @uses $publishedMonographs array List of published monographs in this series
+ * @uses $featuredMonographIds array List of featured monograph IDs in this series
+ * @uses $newReleasesMonographs array List of new monographs in this series
+ *}
+{include file="frontend/components/header.tpl" pageTitle="plugins.block.browse.series"}
+
+<div class="page page_catalog_seriesIndex">
+
+	{* Breadcrumb *}
+	{include file="frontend/components/breadcrumbs_catalog.tpl" type="series" currentTitleKey="plugins.block.browse.series"}
+
+	<h2>
+		{translate key="plugins.block.browse.series"}
+	</h2>
+
+	{* Index with series *}
+	<div class="seriesIndex">
+	<ul>
+
+		{* Series *}
+		{if $browseSeriesFactory && $browseSeriesFactory->getCount()}
+
+		{iterate from=browseSeriesFactory item=browseSeriesItem}				
+		<li>
+		
+		<div class="imageDescription">		
+
+	{* Image and description *}
+
+			<div class="cover"> <a href="{url router=$smarty.const.ROUTE_PAGE page="catalog" op="series" path=$browseSeriesItem->getPath()|escape}">
+			<img src="{url router=$smarty.const.ROUTE_PAGE page="catalog" op="thumbnail" type="series" id=$browseSeriesItem->getId()|escape}" alt="{$browseSeriesItem->getLocalizedFullTitle()|escape}" /></a>
+			</div>
+
+			<div class="metadata">				
+			<h3><a href="{url router=$smarty.const.ROUTE_PAGE page="catalog" op="series" path=$browseSeriesItem->getPath()|escape}"> {$browseSeriesItem->getLocalizedFullTitle()|escape}</a></h3>
+		
+			<div class="description">{$browseSeriesItem->getLocalizedDescription()|strip_unsafe_html|truncate:800}</div>
```

Comment:
Let's not truncate this string. It will cut it off mid-word. Instead, we prefer to let each journal decide the length that is appropriate by adjusting the description.

Reply:
Hmmmm, it is a crude solution I agree. With 800 characters you can convey your message. I had a series that insisted to list the whole scientific committee. This may look okay on the series page but not on a page that tries to provide a quick overview.
Perhaps the solution with announcements could work here: a short version and and a long version.
For the time being I remove the truncation.